pkg/services/serviceaccounts/manager/service.go

```diff
@@ -25,6 +25,10 @@ import (
 const (
 	metricsCollectionInterval = time.Minute * 30
 	defaultSecretScanInterval = time.Minute * 5
+
+	// globallyHideAPIKeysTab is a global setting that hides the apikeys tab
+	// if there are no apikeys present in db or hideapikeys already set
+	GloballyHideAPIKeysTabOrgID int64 = -1
```

Comment:
@alexanderzobnin you think -1 makes sense? i think it is okay. we do not use it elsewhere and this is only used together with the key "hideApikeysTab"